### PR TITLE
Add log for portal service

### PR DESCRIPTION
**Pull request by XiaoxuHU**
This PR is [BUG 1744 AKS Resource cannot open D&S blade in Mooncake - Add Alert whenever Diag Solve Homepage doesnt load](https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_workitems/edit/1744/) 

I add three events(**PortalIFrameLoadingException/PortalIFrameLoadingSuccess/PortalIFrameLoadingStart**) into kusto logging so we can get overall sessions opening iFrame, sessions open successfully, sessions have exception when opening iFrame

### CODE REVIEW

File: AngularApp/projects/app-service-diagnostics/src/app/startup/services/portal.service.ts [hunk 1]
```diff
@@ -152,24 +158,14 @@ export class PortalService {
...
-
     private iframeReceivedMsg(event: Event): void {
         if (!event || !event.data || event.data.signature !== this.portalSignature) {
             return;
         }
 
         const data = event.data.data;
         const methodName = event.data.kind;
-
+        console.log(`portal.service, ${methodName}`);
```
Comment (cindylovescoding) [inline AngularApp/projects/app-service-diagnostics/src/app/startup/services/portal.service.ts:168]: Let's not logging it as console logs

Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/startup/services/portal.service.ts:168]: Yeah, I forget to remove it...

**Review by ShekharGupta1988 — APPROVED** [review]

File: AngularApp/projects/app-service-diagnostics/src/app/startup/services/portal.service.ts [hunk 2]
```diff
@@ -152,16 +158,6 @@ export class PortalService {
         this.postMessage(Verbs.setDirtyState, JSON.stringify(dirty));
     }
 
-    logMessage(level: LogEntryLevel, message: string, ...restArgs: any[]) {
```
[the remark this answers is not in the document]
Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/startup/services/portal.service.ts:155]: Ok, I can keep **logMessage** and also for exceptions I added I can log into action as well as  entry-level

[the remark this answers is not in the document]
Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/startup/services/portal.service.ts:155]: I updated code so for now all logging in portal service will go into both action and entry level

**Review by cindylovescoding — APPROVED** [review]

**Pull request merged**